### DISCUSSION TITLE
Support for SVG Text tag

### DISCUSSION THREAD

**Discussion by asultan123** [ManimCommunity/manim#2819] [Suggestions and Proposals]
## Description of proposed feature
During svg parsing in svg_mobject.py svg <Text></Text> fields are ignored because they fail to match any of the currently supported tag names https://github.com/ManimCommunity/manim/blob/d3bdefbae26cf2fa07750cddc9ff4a0de5c9b7c7/manim/mobject/svg/svg_mobject.py#L204-L235

## How can the new feature be used?
When exporting svg objects for block diagrams from flowcharting software like visio and draw.io it would be helpful to preserve labels given to drawn objects while animating exported svg output in manim.

**Comment by AlexisTercero55** [comment]
Im using Inskape for my svg drawings, I had the same problem, the only way I found for insert text on manim scene is importing from: https://maketext.io/.

![CreateSVG_ManimCE_v0 17 2](https://user-images.githubusercontent.com/87354316/209568009-d35ffccd-a38b-4dec-a825-354644fcdd19.png)

**Reply by naveen521kk** [reply]
> Im using Inskape for my svg drawings

If you're using Inkscape, select your text and choose `Path > Object to Path`, that should convert the text into an SVG path which manim can import. Note, using this will make the text uneditable, make a backup of the original SVG file.
